=================
Amplicon analyses
=================

Marker gene amplicons
---------------------
* 16s rRNA, 18s rRNA, ITS1/2
* Taxonomic profiling
* OTU clustering 
* taxonomy -> function inferences

Links to resources that cover several for the above topics
----------------------------------------------------------
* `Hsiao tutorials <http://bioinformatics.ca/metagenomics2015module2pptx>`_
* `Morgan Langille <https://www.dropbox.com/s/e8e6p62k92s1fo9/Langille_PICRUSt_STAMP_Lecture.pptx?dl=1>`_
* `EDAMAME course <www.edamamecourse.org>`_

Learning objectives
--------------------
* Define a marker gene and its taxonomic relevance.
* Distinguish between amplicon vs. shotgun approach.
* Apply a marker gene analysis workflow
* Analyse the taxonomic composition of an environment.
* Interpret results in biological context
* Describe and execute an amplicon workflow
* Install software for amplicon analysis
* Assemble paired-end reads
* Execute a shell script to automate a process
* Describe input and output files for amplicon workflows and scripts
* Describe the structure and components of a good mapping/metadata file
* Move sequences from compute resources to local computer
* Obtain summary information about sequence files (fasta, fna, fastq)
* Define operational taxaonomic units (OTUs)
* Align sequences, assign taxonomy, and build a tree with representative sequences from OTU definitions
* Calculate similarity of two samples (similarity matrices)
* Visualize comparative diversity across a priori categorical groups
* Convert .biom formatted OTU tables to text files for use outside of QIIME
