================
Shotgun Analyses
================

Shotgun metagenomics & metatranscriptomics
------------------------------------------
* To assemble or not?
* To gene call or not? 
* Kmers (“reference-free”)
* Taxonomic profiling
* Coverage estimation
* Reference mapping / alignment
* Gene Catalogues
* Marker-Based approaches
* Read-based approaches
* Functional profiling/annotation
* Functional hierarchies / ontologies	
* Depth comparisons
* Binning
* Whole genome assembly and evaluation
* Pathway Analysis / Metabolic modeling
* Cross assembly

Links to resources that cover several for the above topics
----------------------------------------------------------

* `John Parkinson <http://bioinformatics.ca/metagenomics2015module5pptx>`_
* `Reference Tutorial <http://www.nbic.nl/uploads/media/Day2_deJager_MG_RAST_workshop_2013.pdf http://www.nbic.nl/uploads/media/MGRast_workshop_130207.pdf>`_
* `BioMAS <https://www.dropbox.com/s/ijwkd4ifgb9nlb2/BioMaS_demo_Recas.mov?dl=0>`_
* `Morgan Langille metagenomic tutorial <http://bioinformatics.ca/metagenomics2015module3pptx>`_
* `Metagenomics Crash Course <http://metag-crash-course.readthedocs.org/en/latest/>`_
* `CABBIO <http://tbb.bio.uu.nl/dutilh/CABBIO/>`_
* `TGAC <https://github.com/TGAC/361Division/blob/master/Metagenomics%202015/TGAC_metagenomics_151015_publicdataresources_JP.pdf>`_

Learning objectives
-------------------

Shotgun metagenomics & metatranscriptomics (Intro):

* Discern the difference between genetic potential and expressed genes.
* Evaluate the advantages and disadvantages between the metaT and metaG
* Aware of the cost and depth of sequencing differences for metaT and metaG

Reference mapping / alignment:

* Understand approaches to the elimination of host-associated or contaminating material
* Apply reference mapping to assess community structure
* Interpret the mapping outcomes as representative abundance of taxa.
* Compare the mapping approach with the "reference-free".

Taxonomic profiling:

* Compare taxonomic profiling using metagenomics and amplicon-based approaches.
* Utilise both marked based and all-read methods for determining taxonomic profiles from metagenomic data
* Apply one or more methods to create a taxonomic profile
* Distinguish reference taxonomies and database limitations.
* Apply appropriate statistical methods for comparative analysis.
* Demonstrate knowledge of the limitations of quantifying microbes as relative abundances

Functional profiling:

* Recognise the role and application of different reference databases in functional assignments
* Use tools to perform functional annotations of nucleic acid and protein sequences found in metagenomics datasets.
* Demonstrate an understanding of the relationship between ontologies (e.g. GO) and functional assignments as a means  of hierarchically viewing the data.
* Critically assess the functional assignments in terms of confidence, with clear understanding about the limitations of the algorithms and databases used. 
* Apply pathway gap-filling methods to overcome low coverage sequencing

