Introduction to RNA-seq analysis 2014
=====================================

Keywords
--------

Alignment, Differential-expression, Feature-summarisation,
Pre-processing, QC

Authors
-------

Frederik Coppens (@frcop)

Type
----

-  Lecture

--------------

`Top <#sub-module-title>`__ \| `Keywords <#keywords>`__ \|
`Authors <#authors>`__ \| `Type <#type>`__ \|
`Description <#description>`__ \| `Aims <#aims>`__ \|
`Prerequisites <#prerequisites>`__ \| `Target
audience <#target-audience>`__ \| `Learning
objectives <#learning-objectives>`__ \| `Materials <#materials>`__ \|
`Data <#data>`__ \| `Timing <#timing>`__ \| `Content
stability <#content-stability>`__ \| `Technical
requirements <#technical-requirements>`__ \| `Literature
references <#literature-references>`__

--------------

Description
-----------

This lecture gives an overview how to perform an RNA-seq experiment. A
general RNA-seq workflow is outlined when a good quality genome sequence
is available for your species.

Aims
----

The aim is that participants are aware of the general steps in an
RNA-seq experiment and are able to identify appropriate tools for each
step. For each step the most important features, some tools and things
to be aware of are listed.

Prerequisites
-------------

-  None

Target audience
---------------

beginner, biologist

Learning objectives
-------------------

-  list steps of an RNA-seq analysis
-  discuss QC of raw data
-  decide which alignment approach is appropriate for your use case
-  discuss quality of alignment
-  recognise the importance of appropriate statistical methods for
   differential expression

--------------

`Top <#sub-module-title>`__ \| `Keywords <#keywords>`__ \|
`Authors <#authors>`__ \| `Type <#type>`__ \|
`Description <#description>`__ \| `Aims <#aims>`__ \|
`Prerequisites <#prerequisites>`__ \| `Target
audience <#target-audience>`__ \| `Learning
objectives <#learning-objectives>`__ \| `Materials <#materials>`__ \|
`Data <#data>`__ \| `Timing <#timing>`__ \| `Content
stability <#content-stability>`__ \| `Technical
requirements <#technical-requirements>`__ \| `Literature
references <#literature-references>`__

--------------

Materials
---------

-  `Slides in
   pdf <Content/RNA-Seq/Frederik_Coppens/2014_RNAseq_intro/NGS_introduction_3_Data_analysis.pdf>`__.

Data
----

-  not applicable

--------------

`Top <#sub-module-title>`__ \| `Keywords <#keywords>`__ \|
`Authors <#authors>`__ \| `Type <#type>`__ \|
`Description <#description>`__ \| `Aims <#aims>`__ \|
`Prerequisites <#prerequisites>`__ \| `Target
audience <#target-audience>`__ \| `Learning
objectives <#learning-objectives>`__ \| `Materials <#materials>`__ \|
`Data <#data>`__ \| `Timing <#timing>`__ \| `Content
stability <#content-stability>`__ \| `Technical
requirements <#technical-requirements>`__ \| `Literature
references <#literature-references>`__

--------------

Timing
------

2h

Content stability
-----------------

Stable

Technical requirements
----------------------

-  `FastQC <http://www.bioinformatics.babraham.ac.uk/projects/fastqc/>`__
-  `FastX toolkit <http://hannonlab.cshl.edu/fastx_toolkit/>`__
-  `Cutadapt <https://code.google.com/p/cutadapt/>`__
-  `Trimmomatic <http://www.usadellab.org/cms/?page=trimmomatic>`__
-  `GSNAP <http://research-pub.gene.com/gmap>`__
-  `Bowtie <http://bowtie-bio.sourceforge.net/index.shtml>`__
-  `TopHat <http://tophat.cbcb.umd.edu>`__
-  `Samtools <http://www.htslib.org>`__
-  `HTseq <http://www-huber.embl.de/users/anders/HTSeq/doc/overview.html>`__
-  `edgeR <http://bioconductor.org/packages/release/bioc/html/edgeR.html>`__
-  `DESeq <https://bioconductor.org/packages/release/bioc/html/DESeq.html>`__
-  `Cufflinks <http://cole-trapnell-lab.github.io/cufflinks/>`__

Literature references
---------------------

-  to do

Changelog
---------

-  2015-08-25: Added links to tools
-  2015-02-19: Upload to gitlab

Comments
--------

-  I did not check if the use of all figures is allowed or properly
   acknowledged.
-  A license needs to be added

--------------

`Top <#sub-module-title>`__ \| `Keywords <#keywords>`__ \| `Authors <#authors>`__ \| `Type <#type>`__ \| `Description <#description>`__ \| `Aims <#aims>`__ \| `Prerequisites <#prerequisites>`__ \| `Target audience <#target-audience>`__ \| `Learning objectives <#learning-objectives>`__ \| `Materials <#materials>`__ \| `Data <#data>`__ \| `Timing <#timing>`__ \| `Content stability <#content-stability>`__ \| `Technical requirements <#technical-requirements>`__ \| `Literature references <#literature-references>`__
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
